### PR TITLE
Callbacks support

### PR DESCRIPTION
**Pull request by onerinas**
This addresses the below error while using callbacks:

``` 
- utils - ERROR - Exception occured: Callbacks are not supported by "cordova.plugins.iosrtc.getUserMedia" anymore, use Promise instead.
detectDeprecatedCallbaksUsage
```

### CODE REVIEW

**Comment by hthetiot** [comment]
@onerinas I dont understand why you fork.
Callback have been added again on 5.0.5 and 6.0.2 and 6.0.3 implement M69 Native RTCPeerConnection.(addTrack|removeTrack) and sdpSemantics unified-plan support

**Comment by hthetiot** [comment]
https://github.com/cordova-rtc/cordova-plugin-iosrtc/commit/1a395c576bf98bfb12da705899396c50813a67cc#diff-a319de570361194065829caae50e0574

**Comment by hthetiot** [comment]
Instead of fixing in your corner what is already fixed on original repository you can help and make PR.

**Comment by hthetiot** [comment]
6.0.4 will be release tomorrow and 5.0.5 and 6.0.3 contains new fixes.
- https://github.com/cordova-rtc/cordova-plugin-iosrtc/blob/master/CHANGELOG.md#version-604

File: js/iosrtc.js [hunk 1]
```diff
@@ -191,16 +191,15 @@ function registerGlobals(doNotRestoreCallbacksSupport) {
...
-	navigator.webkitGetUserMedia            = getUserMedia;
-	navigator.mediaDevices.getUserMedia     = getUserMedia;
-	navigator.mediaDevices.enumerateDevices = enumerateDevices;
-
 	// Restore Callback support
 	if (!doNotRestoreCallbacksSupport) {
 		restoreCallbacksSupport();
 	}
 
+	navigator.getUserMedia                  = getUserMedia;
```
Comment (onerinas) [inline js/iosrtc.js:190]: @hthetiot callbacks support wasn't working for me when I was trying to use. Had to make this change to make it work. I'll try to check the versions you mentioned and create a PR if this change is not there in the repository.

Comment (onerinas) [reply js/iosrtc.js:190]: Looks like we still set [`navigator.getUserMedia`](https://github.com/cordova-rtc/cordova-plugin-iosrtc/blob/6ab6f8258d59b667382293ba08bd502e351c0730/js/iosrtc.js#L185) before calling [`restoreCallbacksSupport();`](https://github.com/cordova-rtc/cordova-plugin-iosrtc/blob/6ab6f8258d59b667382293ba08bd502e351c0730/js/iosrtc.js#L192)


Will create the PR.

**Comment by onerinas** [comment]
Moved to https://github.com/cordova-rtc/cordova-plugin-iosrtc/pull/453

**Pull request closed without merging**